gensim/models/phrases.py

```diff
@@ -83,16 +83,20 @@
 import functools as ft
 import itertools as it
 from math import log
-from inspect import getargspec
 import pickle
 import six
 
-from six import iteritems, string_types, next
+from six import iteritems, string_types, PY2, next
 
 from gensim import utils, interfaces
 
 logger = logging.getLogger(__name__)
 
+if PY2:
+    from inspect import getargspec
```

Comment:
you can make identical import like

`from inspect import getargspec` and
`from inspect import getfullargspec as getargspec`

And you no need to change something after